README.md

```diff
@@ -6,8 +6,57 @@ spec](https://github.com/singer-io/getting-started/blob/master/SPEC.md).
+  - [Bidding Strategies](https://developers.google.com/google-ads/api/reference/rpc/v10/BiddingStrategy)
+  - [Accessible Bidding Strategies](https://developers.google.com/google-ads/api/reference/rpc/v10/AccessibleBiddingStrategy)
+  - [Reporting](https://developers.google.com/google-ads/api/docs/reporting/overview)
+    - [Age Range Performance Report](https://developers.google.com/google-ads/api/fields/v10/age_range_view)
+    - [Campaign Performance Report](https://developers.google.com/google-ads/api/fields/v10/campaign)
+    - [Campaign Audience Performance Report](https://developers.google.com/google-ads/api/fields/v10/campaign_audience_view)
+    - [Call Metrics Call Details Report](https://developers.google.com/google-ads/api/fields/v10/call_view)
+    - [Click Performance Report](https://developers.google.com/google-ads/api/fields/v10/click_view)
+    - [Display Keyword Performance Report](https://developers.google.com/google-ads/api/fields/v10/display_keyword_view)
+    - [Display Topics Performance Report](https://developers.google.com/google-ads/api/fields/v10/topic_view)
+    - [Gender Performance Report](https://developers.google.com/google-ads/api/fields/v10/gender_view)
+    - [Geo Performance Report](https://developers.google.com/google-ads/api/fields/v10/geographic_view)
+    - [User Location Performance Report](https://developers.google.com/google-ads/api/fields/v10/user_location_view)
+    - [Keywordless Query Report](https://developers.google.com/google-ads/api/fields/v10/dynamic_search_ads_search_term_view)
+    - [Keywords Performance Report](https://developers.google.com/google-ads/api/fields/v10/keyword_view)
+    - [Landing Page Report](https://developers.google.com/google-ads/api/fields/v10/landing_page_view)
+    - [Expanded Landing Page Report](https://developers.google.com/google-ads/api/fields/v10/expanded_landing_page_view)
+    - [Placeholder Feed Item Report](https://developers.google.com/google-ads/api/fields/v10/feed_item)
+    - [Placeholder Report](https://developers.google.com/google-ads/api/fields/v10/feed_placeholder_view)
+    - [Placement Performance Report](https://developers.google.com/google-ads/api/fields/v10/managed_placement_view)
+    - [Search Query Performance Report](https://developers.google.com/google-ads/api/fields/v10/search_term_view)
+    - [Shopping Performance Report](https://developers.google.com/google-ads/api/fields/v10/shopping_performance_view)
+    - [UserLocation Performance Report](https://developers.google.com/google-ads/api/fields/v10/user_location_view)
+    - [Video Performance Report](https://developers.google.com/google-ads/api/fields/v10/video)
+    - Account Performance Report
+    - [Ad Group Performance Report](https://developers.google.com/google-ads/api/fields/v10/ad_group)
+    - [Ad Group Audience Performance Report](https://developers.google.com/google-ads/api/fields/v10/ad_group_audience_view)
+    - Ad Performance Report
```

Comment:
Why link is not added for Account Performance Report and Ad Performance Report

Reply:
Because we didn't find any link for the same in the docs

Reply:
Because we didnt find any link for the same in the API documentation

Reply:
I think we should separate core streams and reports, then ensure that we've alphabetized to make it easier to read and maintain.

The ad performance report just queries the ad resource, so the same documentation link can be used.

Account performance report queries the customer resource so that same documentation link can be used.

Reply:
Updated according to the siggestion